jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/util/AbstractView.java

```diff
@@ -31,4 +31,7 @@ protected native void tooltip(final HTMLElement e) /*-{
         $wnd.jQuery(e).tooltip();
     }-*/;
 
+    protected native void popover(final HTMLElement e) /*-{
```

Comment:
I end up creating a component for the popover on this PR: https://github.com/kiegroup/jbpm-wb/pull/652/files#diff-85ccc65888795800cb45c1d405460fadR28 Perhaps you can reuse it.